WooCommerce/Classes/ViewRelated/Notifications/ReviewsDataSource.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import UIKit
+import Yosemite
+
+
+/// Abstracts the datasource used to render the Product Review list
+protocol ReviewsDataSource: UITableViewDataSource, UITableViewDelegate {
+
+    /// Boolean indicating if there are reviews
+    ///
+    var isEmpty: Bool { get }
+
+    /// Identifiers of the Products mentioned in the reviews.
+    /// Guaranteed to be uniqued (does not contain duplicates)
```

Comment:
Yes, but returning an array we can guarantee it is sorted, which to be honest and more importantly, we ca pass it directly to the API (all stores expect arrays instead of sets, as far as I recall)

Reply:
> we ca pass it directly to the API (all stores expect arrays instead of sets, as far as I recall)

this feels like an API implementation detail to me, where we can convert the set to an array at the Store/Networking level?